modules/calculators/Pid.h

```diff
@@ -0,0 +1,65 @@
+/**
+ * @file Pid.h
+ * @brief PIDを計算するクラス
+ * @author miyahara046
+ */
+
+#ifndef PID_H
+#define PID_H
+
+//   PIDゲインを保持する構造体
```

Comment:
```suggestion
// PIDゲインを保持する構造体
```
コメントの空白は半角スペース1個が通例な気がする。
コメントだから、人間が見えればどうだっていいんだけどね。